### PR TITLE
Added dark CSS and JS

### PR DESCRIPTION
**Pull request by fronbasal**
This is a proposal to add a dark theme to the website.

I often utilize your website at night, so a dark theme would be great.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,43 +1,53 @@
...
+		<textarea id="input" rows="4" placeholder="Paste curl here"></textarea>
+		<div id="output"></div>
+	</main>
+
+
+	<footer>
+		&copy;
+		<script>
+			document.write(new Date().getFullYear());
+		</script> Matt Holt - <a href="https://github.com/mholt/curl-to-go">View on GitHub</a> - <a href="#" id="dark-trigger">dark mode</a>
```
Comment (mholt) [inline index.html]: Change the href to "javascript:"

**Review by mholt — CHANGES_REQUESTED** [review]
Thanks for this too, same changes as before: change the href and remove all the other changes that aren't adding the dark theme ;)

**Comment by fronbasal** [comment]
Hey! I also did the changes to this project. Can you check again?

**Review by mholt — APPROVED** [review]

**Comment by mholt** [comment]
Thanks!

**Pull request merged**